src/OnboardingSPA/pages/Steps/GetStarted/SiteTypeSetup/SecondarySite/index.js

```diff
@@ -29,154 +29,219 @@ const StepPrimarySetup = () => {
 		setIsDrawerSuppressed( true );
 		setDrawerActiveView( VIEW_NAV_GET_STARTED );
 		setIsHeaderNavigationEnabled( true );
+		getSiteClassificationsData();
 	}, [] );
 
-	const [ clickedIndex, changeCategory ] = useState( -1 );
+	const contents = getContents();
+	const defaultPrimaryType = 'business';
+	const [ siteClassData, setSiteClassData ] = useState();
```

Comment:
Please consider some of the `PrimaryType` comments for this file as well wherever relevant.